### PR TITLE
[zh]Update content/zh/docs/concepts/workloads/controllers/job.md

### PR DESCRIPTION
**Pull request by zyy19981018**

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ It takes around 10s to complete.
 ```shell
 kubectl apply -f https://kubernetes.io/examples/controllers/job.yaml
 ```
-
+The output is similar to this:
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/job.md:69]: 这些英文要用 `<!-- -->` 括起来。参见其他文档的翻译示例。

**Comment by howieyuen** [comment]
please squash commits into one

**Comment by JarHMJ** [comment]
related issues: #33476 concepts-5

**Pull request merged**